conda_rpms/tests/unit/build_rpm_structure/test_create_rpmbuild_content.py

```diff
@@ -0,0 +1,110 @@
+import os
+import unittest
+
+from conda_gitenv import manifest_branch_prefix
+import conda_gitenv.tests.integration.setup_samples as setup_samples
+
+from conda_rpms.build_rpm_structure import create_rpmbuild_content
+import conda_rpms.tests as tests
+
+
+class Test(tests.CommonTest):
+    def setUp(self):
+        self.repo = setup_samples.create_repo('conda-rpms-basic')
+        self.bname = 'default'
+        env_spec = """
+                   channels:
+                       - defaults
+                   env:
+                       - python
+                   """
+        # Require to create a dummy manifest branch.
+        self.repo.create_head(manifest_branch_prefix + self.bname)
+        self.branch = setup_samples.add_env(self.repo, self.bname, env_spec)
+        func = 'conda_rpms.build_rpm_structure.create_rpmbuild_for_tag'
+        self.mock_create_tag = self.patch(func)
+        func = 'conda_rpms.generate.render_env'
+        self.mock_render_env = self.patch(func, return_value='dummy-env')
+        self.config = dict(rpm=dict(prefix='prefix'))
+
+    def _add_label(self, fname, tag):
+        self.branch.checkout()
+        dname = os.path.join(self.repo.working_dir, 'labels')
+        if not os.path.isdir(dname):
+            os.makedirs(dname)
+        fpath = os.path.join(dname, fname)
+        with open(fpath, 'w') as fo:
+            fo.write(tag)
+        self.repo.index.add([fpath])
+        comment = 'Add label {}'.format(fname)
+        self.repo.index.commit(comment)
```

Comment:
I wonder whether it's worth putting this code somewhere so that it can be reused in other tests? It just seems like another test may require this sort of thing,